src/Components/CollectionCard/CollectionCardModal.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import Bin from '../../assets/icons/Bin.png';
+import Tag from '../../assets/icons/price-tag 1.png';
+import Star from '../../assets/icons/Star1.png';
+import Close from '../../assets/icons/Close.png';
+import Img from '../../assets/icons/image 22.png';
+
+export default function CollectionCardModal({ card, type }) {
+        
+      return (
+        <>
```

Comment:
why is there double fragments here, please remove them and maybe add one div instead